document_test.go

```diff
@@ -1372,3 +1373,24 @@ func TestDocument_GetEndOfLinePosition(t *testing.T) {
 		t.Errorf("Should be %#v, got %#v", ex, ac)
 	}
 }
+
+func TestCharColumnWidth(t *testing.T) {
+	var countCharTests = []struct {
+		text          string
+		expectedCount int
+	}{
+		{"", 0},
+		{"a", 1},
+		{"あ", 2},
+		{"け", 2},
+		{"く", 2},
+	}
+
+	for _, test := range countCharTests {
```

Comment:
can this be simplified to just this?
```
for _, test := range countCharTests {
  require.Equal(t, test.expectedCount, runewidth.StringWidth(test.text))
}
```

Reply:
Yes, sure. The above is just me pressing tab on copilot